packages/catalog-model/src/entity/policies/FieldFormatEntityPolicy.test.ts

```diff
@@ -110,4 +114,9 @@ describe('FieldFormatEntityPolicy', () => {
     data.metadata.tags.push('Hello World');
     await expect(policy.enforce(data)).rejects.toThrow(/tags.*"Hello World"/i);
   });
+
```

Comment:
I'd like these tests to be very detailed and to exercise every single possible variation :)

- accepts missing links
- accepts empty links array
- rejects bad links value

- rejects missing links.url
- rejects empty links.url value
- rejects bad links.url value

- accepts missing links.label
- rejects empty links.label value
- rejects bad links.label value

- accepts missing links.icon
- rejects empty links.icon value
- rejects bad links.icon value

Reply:
Done